README.md

```diff
@@ -23,16 +23,49 @@ For DevOps or SRE teams this charm will make operating Synapse simple and
 straightforward through Juju's clean interface. It will allow easy deployment
 into multiple environments for testing of changes.
 
-## Project and community
+## Get started
+
+To begin, refer to the [Getting Started](https://charmhub.io/synapse/docs/tutorial-getting-started)
+tutorial for step-by-step instructions.
+
+### Basic operations
+
+#### Configure a server name
+
+The configuration `server_name` sets the public-facing domain of the server and
+refers to [`server_name`](https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html#server_name) Synapse configuration.
+
+To change it to `tutorial-synapse.juju.local`, for example, run the following
+command:
+
+```
+juju config synapse server_name=tutorial-synapse.juju.local
+```
+
+#### Create a user
 
-The Synapse Operator is a member of the Ubuntu family. It's an open source
-project that warmly welcomes community projects, contributions, suggestions,
-fixes and constructive feedback.
-* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
-* [Get support](https://discourse.charmhub.io/)
-* [Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
-* [Contribute](https://charmhub.io/synapse/docs/contributing)
-* [Getting Started](https://charmhub.io/synapse/docs/getting-started)
-Thinking about using the Synapse Operator for your next project? [Get in touch](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)!
-
----
+The following command creates a local user named `alice`.
+
+```
+juju run-action synapse/0 register-user username=alice password=<secure-password> admin=no
```

Comment:
Does `juju run-action` work or is it `juju run` now?

Reply:
Ahhh always forget that..changing now! Thanks :)